src/common/KokkosKernels_Handle.hpp

```diff
@@ -209,6 +220,10 @@ class KokkosKernelsHandle
   typedef
     typename KokkosSparse::Experimental::SPTRSVHandle<const_size_type, const_nnz_lno_t, const_nnz_scalar_t, HandleExecSpace, HandleTempMemorySpace, HandlePersistentMemorySpace>
       SPTRSVHandleType;
+#ifdef KOKKOSKERNELS_ENABLE_SUPERNODAL_SPTRSV
+  typedef
```

Comment:
Prefer C++11 `using` syntax to `typedef`.
```c++
using integer_view_host_t = typename SPTRSVHandleType::integer_view_host_t;
```

Reply:
We switched to "using"!!